app/Http/Controllers/CourseController.php

```diff
@@ -153,7 +160,7 @@ public function getCourseDetails(Request $request)
 		$associatedCourse = $user->enrolledIn()->where('course_id', $course->id)->first();
```

Comment:
Cuál es la lógica detrás de esta validación aquí? Queremos hacerlo en un `getCourseDetails` function?

Reply:
Non lo so. Esta función y la de `associate()` no fueron creadas ni son utilizadas por este user story (creo que @rubendelapena o @katiearriagam podrían saber más sobre ellas).

Reply:
@katiearriagam dice el git blame que tu agregaste la linea hahaha want to weigh in?

Reply:
Queremos validar si el usuario ya está asociado al curso que está tratando de agregar para decirle esto y poder cerrar el flujo para asociar un student con un course. 

No entendí la pregunta de si queremos hacerlo en un `getCourseDetails`. Estamos tratando de mostrar los course details del curso que estamos tratanto de agregar. Maybe un mejor nombre sería `getCandidateCourseDetails`? Pero no sé si ésa sea tu pregunta.

![image](https://user-images.githubusercontent.com/14140121/54380461-2810dd80-4651-11e9-8190-db60feff0cdf.png)

Reply:
Katie: tienes razón.

Reply:
Esto está muy bien.

Reply:
Amazing pr